data.json

```diff
@@ -9,5 +9,15 @@
       "linkedin": "https://www.linkedin.com/in/rahulbansal52/",
       "quote": "Open source is 💓."
     }
+  ]
+       "data": [
+      "name": {Akash Maurua},
+      "batch": {YOUR_BATCH_COMMENCEMENT_YEAR},
```

Comment:
Please learn about `JSON` format. you can refer [this](https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Objects/JSON)